crates/telio-firewall/src/firewall.rs

```diff
@@ -922,7 +988,9 @@ impl StatefullFirewall {
                 }
                 is_in_cache
             }
-            IcmpErrorKey::Udp(Some(udp_key)) => {
+            IcmpErrorKey::Udp(Some(link)) => {
```

Comment:
Could you also rename `is_port_whitelisted` to `is_conn_whitelisted` in TCP too? For the same reason.

Reply:
Done